src/lib/input/autosize.ts

```diff
@@ -36,21 +49,37 @@ export class MdTextareaAutosize implements AfterViewInit {
 
   constructor(private _elementRef: ElementRef) { }
 
-  /** The minimum height of the textarea as determined by minRows. */
-  get _minHeight() {
-    return this.minRows ? `${this.minRows * this._cachedLineHeight}px` : null;
+  /** Sets the minimum height of the textarea as determined by minRows. */
+  _setMinHeight(): void {
+    const minHeight = this.minRows && this._cachedLineHeight
```

Comment:
Prefer putting the `?` at the end of the previous line rather than the start of the new line